Remotion/Web/Development.WebTesting.ControlObjects/WebTreeViewNodeControlObject.cs

```diff
@@ -112,7 +113,20 @@ public WebTreeViewNodeControlObject WithDisplayTextContains (string containsDisp
             CssComparisonOperator.SubstringMatch,
             DiagnosticMetadataAttributes.Content,
             containsDisplayText);
-        return new WebTreeViewNodeControlObject (_webTreeViewNode.Context.CloneForControl (nodeScope));
+
+        return CreateWebTreeViewNodeControlObject (nodeScope);
+      }
+
+      private WebTreeViewNodeControlObject CreateWebTreeViewNodeControlObject (ElementScope nodeScope)
+      {
+        try
+        {
+          return new WebTreeViewNodeControlObject (_webTreeViewNode.Context.CloneForControl (nodeScope));
+        }
+        catch (StaleElementException)
+        {
+          throw new WebTestException ("This element has been removed from the DOM.");

```

Comment:
Pass the original exception?

Reply:
Tracked in https://github.com/re-motion/Trunk-SVN2Git/pull/42#discussion_r357616471